ably/util/exceptions.py

```diff
@@ -63,6 +63,10 @@ def from_exception(e):
             return e
         return AblyException("Unexpected exception: %s" % e, 500, 50000)
 
+    @staticmethod
+    def from_dict(d):
```

Comment:
We can use `err` or `error` as the variable name since that's the only fit for this method. Or say `value`.

Reply:
good idea, i've changed it to `value` now